package a40Exception_Handling_Part2_43;

import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.File;
import java.io.FileReader;
import java.io.FileWriter;

public class bProgram1 {
	BufferedReader br;
	
public static void main(String[] args) throws Exception {
		
		File file = new File("./sample2.txt");
		if(!file.exists())
			file.delete();
			file.createNewFile();
		
		try(
			BufferedReader br = new BufferedReader(new FileReader(file));
			BufferedWriter bw = new BufferedWriter(new FileWriter(file));
				){
		String existingText = new String();
		String line = "";
		
		while((line = br.readLine()) != null) {
			existingText = existingText + line + "\n";
		}

		String s = "What are you doing";
		
		bw.write(existingText + s);
		//bw.write(s.toCharArray());
		}catch(Exception ae) {
			System.out.println(ae.printStackTrace());
		}
	}

}
